js/layout-javascript/agenda-code.js

```diff
@@ -590,23 +655,32 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
     })
-    .on('click', '.agenda-date-selector li', function() {
-      // prevents clicking the active one
-      // prevents clicking the placeholder
-      if ($(this).hasClass('active') || $(this).hasClass('placeholder')) {
+    .on('keydown', '.agenda-date-selector li', function(event) {
```

Comment:
Why are we removing `click` from this event listener?

```suggestion
    .on('click keydown', '.agenda-date-selector li', function(event) {
```